src/custom/pages/Faq/index.tsx

```diff
@@ -266,8 +266,8 @@ export default function Faq() {
           </p>
 
           <p>
-            The code has been carefully tested and peer-reviewed. Although this can be seen as a step forward in terms
-            of security, it&#39;s recommended to use the protocol at <strong>your own risk</strong>
+            The code has been carefully tested, peer-reviewed and fully <a href="https://github.com/gnosis/gp-v2-contracts/blob/main/audits/GnosisProtocolV2May2021.pdf"> 
+            audited </a>. Although this can be seen as a step forward in terms of security, it&#39;s recommended to use the protocol at <strong>your own risk</strong>
```

Comment:
```suggestion
            audited</a>. Although this can be seen as a step forward in terms of security, it&#39;s recommended to use the protocol at <strong>your own risk</strong>.
```